### PR TITLE
Some mappings I wanted to do

### PR DESCRIPTION
**Pull request by halotroop2288**
Just making my own life easier

### CODE REVIEW

File: mappings/net/minecraft/FormattedString.mapping [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_241 net/minecraft/FormattedString
+	METHOD method_809 format (Ljava/lang/String;)Ljava/lang/String;
```
Comment (valoeghese) [inline mappings/net/minecraft/FormattedString.mapping:2]: Package?

File: mappings/net/minecraft/achievement/Achievement.mapping [hunk 2]
```diff
@@ -1 +1,27 @@
...
+	FIELD field_1202 xLocation I
+	FIELD field_1203 yLocation I
+	FIELD field_1204 parent Lnet/minecraft/class_24;
+	FIELD field_1205 achievementIcon Lnet/minecraft/class_31;
+	FIELD field_1206 description Ljava/lang/String;
+	FIELD field_1207 challenge Z
+	FIELD field_1208 formattedString Lnet/minecraft/class_241;
+	METHOD <init> (ILjava/lang/String;IILnet/minecraft/class_124;Lnet/minecraft/class_24;)V
+		ARG 1 id
+		ARG 2 descriptionText
```
Comment (valoeghese) [inline mappings/net/minecraft/achievement/Achievement.mapping:11]: Just description is fine imo

File: mappings/net/minecraft/achievement/Achievement.mapping [hunk 3]
```diff
@@ -1 +1,27 @@
 CLASS net/minecraft/class_24 net/minecraft/achievement/Achievement
+	FIELD field_1202 xLocation I
+	FIELD field_1203 yLocation I
```
Comment (valoeghese) [inline mappings/net/minecraft/achievement/Achievement.mapping:3]: xLocation is a bit wordy.

File: mappings/net/minecraft/achievement/Achievements.mapping [hunk 4]
```diff
@@ -1,4 +1,8 @@
 CLASS net/minecraft/class_23 net/minecraft/achievement/Achievements
+	FIELD field_108 lowestXLocation I
```
Comment (valoeghese) [inline mappings/net/minecraft/achievement/Achievements.mapping:2]: Same here. xLocation is unnecessarily wordy

File: mappings/net/minecraft/achievement/Achievement.mapping [hunk 5]
```diff
@@ -1 +1,27 @@
...
+		ARG 2 descriptionText
+		ARG 3 xLocation
+		ARG 4 yLocation
+	METHOD <init> (ILjava/lang/String;IILnet/minecraft/class_31;Lnet/minecraft/class_24;)V
+		ARG 1 id
+		ARG 2 descriptionText
+		ARG 3 xLocation
+		ARG 4 yLocation
+	METHOD method_1040 challenge ()Lnet/minecraft/class_24;
+	METHOD method_1042 formatter (Lnet/minecraft/class_241;)Lnet/minecraft/class_24;
```
Comment (valoeghese) [inline mappings/net/minecraft/achievement/Achievement.mapping:25]: Are these getters? Getters have a get prefix

File: mappings/net/minecraft/achievement/Achievement.mapping [hunk 6]
```diff
@@ -1 +1,27 @@
...
+	METHOD method_1043 getDescription ()Ljava/lang/String;
+	METHOD method_1044 isChallenge ()Z
```
Comment (valoeghese) [inline mappings/net/minecraft/achievement/Achievement.mapping:27]: Did challenges exist back in achievement days? I thought that was added when advancement stuff was added

File: mappings/net/minecraft/stat/Stat.mapping [hunk 7]
```diff
@@ -1,9 +1,20 @@
...
 	FIELD field_2030 TIME_FORMATTER Lnet/minecraft/class_498;
 	FIELD field_2031 DISTANCE_FORMATTER Lnet/minecraft/class_498;
+	METHOD <init> (ILjava/lang/String;)V
+		ARG 1 id
+		ARG 2 name
+	METHOD <init> (ILjava/lang/String;Lnet/minecraft/class_498;)V
+		ARG 1 id
+		ARG 2 name
 	METHOD method_1676 register ()Lnet/minecraft/class_139;
+	METHOD method_1677 orphan ()Lnet/minecraft/class_139;
```
Comment (valoeghese) [inline mappings/net/minecraft/stat/Stat.mapping:18]: Is this a setter or a getter? If a getter name accordingly

File: mappings/net/minecraft/achievement/Achievement.mapping (hunk 5 above)
Comment (halotroop2288) [reply mappings/net/minecraft/achievement/Achievement.mapping:25]: They are builders.

File: mappings/net/minecraft/achievement/Achievement.mapping (hunk 6 above)
Comment (halotroop2288) [reply mappings/net/minecraft/achievement/Achievement.mapping:27]: Yes. They're purely aesthetic. Back then they really didn't have a name. They were just spikey.

File: mappings/net/minecraft/stat/Stat.mapping (hunk 7 above)
Comment (halotroop2288) [reply mappings/net/minecraft/stat/Stat.mapping:18]: Again, a builder.
Context: `Achievements.OPEN_INVENTORY = new Achievement(0, "openInventory", 0, 0, ItemType.book, null).orphan().register();`

File: mappings/net/minecraft/achievement/Achievement.mapping (hunk 5 above)
Comment (valoeghese) [reply mappings/net/minecraft/achievement/Achievement.mapping:25]: Ah, I didn’t look close enough. That’s right then

File: mappings/net/minecraft/FormattedString.mapping (hunk 1 above)
Comment (valoeghese) [reply mappings/net/minecraft/FormattedString.mapping:2]: Maybe move to util packagw

File: mappings/net/minecraft/achievement/Achievement.mapping [hunk 8]
```diff
@@ -1 +1,27 @@
 CLASS net/minecraft/class_24 net/minecraft/achievement/Achievement
+	FIELD field_1202 collumn I
```
Comment (valoeghese) [inline mappings/net/minecraft/achievement/Achievement.mapping:2]: Should be spelt `column`

**Comment by valoeghese** [comment]
Also go into the build.gradke and bump the build number to 14 because we do that manually here

**Comment by valoeghese** [comment]
I can’t request reviews from mobile so
Manual ping time
@shedaniel @liach review thank

File: mappings/net/minecraft/client/model/ModelPart.mapping [hunk 9]
```diff
@@ -0,0 +1,23 @@
+CLASS net/minecraft/class_163 net/minecraft/client/model/ModelPart
+	FIELD field_2292 rotationPointX F
+	FIELD field_2293 rotationPointY F
+	FIELD field_2294 rotationPointZ F
```
Comment (liach) [inline mappings/net/minecraft/client/model/ModelPart.mapping:4]: maybe pivotX/Y/Z?

**Comment by valoeghese** [comment]
Can you address liach's concerns?

**Comment by liach** [comment]
It should be fine as is i guess

**Comment by valoeghese** [comment]
Fine. I'll do it myself :yeefruckinhwa:

**Comment by valoeghese** [comment]
I'll merge this in a few hours (like, 8 hours?)

**Comment by Devan-Kerman** [comment]
8 is infinity on it's side 🚲🏠 time
